app/src/publishers/emailPublisher.js

```diff
@@ -9,7 +9,7 @@ const TEMPLATE_MAP = {
     'forma-alerts': 'forest-change-notification',
     forma250GFW: 'forest-change-notification',
     'terrai-alerts': 'forest-change-notification',
-    'glad-alerts': 'forest-change-notification'
+    'glad-alerts': 'forest-change-notification-no-chart-copy'
```

Comment:
this is the wrong template! It should be the same as before.

Reply:
Sorry it should be `forest-change-notification-glads-{{lang}}`

Reply:
Fixed :+1: